Podfile

```diff
@@ -23,7 +23,7 @@ def shared_with_networking_pods
     pod 'AFNetworking', '3.2.1'
     pod 'Alamofire', '4.7.2'
     pod 'wpxmlrpc', '0.8.3'
-    pod 'WordPressKit', '1.2.1'
+    pod 'WordPressKit', '1.3'
```

Comment:
Argh, sorry for this :) I pushed out my own PR today that updates it to `'1.4.0-beta.1` — this is what's causing the merge conflict. Make sure it still points to `'1.4.0-beta.1` after you resolve the conflict :)